app/src/main/java/space/taran/arknavigator/ui/fragments/ResourcesFragment.kt

```diff
@@ -248,7 +265,10 @@ class ResourcesFragment(val root: Path?, val path: Path?): MvpAppCompatFragment(
 
                 Log.d(RESOURCES_SCREEN, "sorting direction changed, ascending = $newAscending")
 
-                presenter.gridPresenter.updateAscending(newAscending)
+                setProgressVisibility(true, "Sorting")
+                viewLifecycleOwner.lifecycleScope.launch {
+                    presenter.gridPresenter.updateAscending(newAscending)
+                }
```

Comment:
`setProgressVisibility` must be called from the presenter to restore the state when the fragment is recreated.

Can we provide `presenterScope` to a `gridPresenter`, launch `updateSorting` and `updateAscending` 
with it instead of `lifecycleScope`?